controller/workitemtype_blackbox_test.go

```diff
@@ -220,6 +248,10 @@ func CreateWorkItemType(id uuid.UUID, spaceID uuid.UUID) app.CreateWorkitemtypeP
 		Request: &http.Request{Host: "api.service.domain.org"},
 	}
 	spaceSelfURL := rest.AbsoluteURL(reqLong, app.SpaceHref(spaceID.String()))
+
+	// fetch category
+	category := getCategory(db)
```

Comment:
I don't like this. We should be more explicit into which category you want to put a work item type. Why not pass the category itself in to the function instead of the `db` object?